README.md

```diff
@@ -69,6 +69,10 @@ No SSL vhosts are configured by default, but you can add them using the same pat
         certificate_chain_file: "/path/to/certificate_chain.crt"
       }
 
+If you would like to only create vHosts when the certificate is available (e.g. when using Let’s encrypt), set
```

Comment:
As the new var is meant to be a top-level variable, please move this documentation down below the existing SSL vhosts documentation, like:

```
    apache_ignore_missing_ssl_certificate: true

If you would like to only create SSL vhosts when the vhost certificate is present (e.g. when using Let’s Encrypt), set `apache_ignore_missing_ssl_certificate` to `false`. When doing this, you might need to run your playbook more than once so all the vhosts are configured (if another part of the playbook generates the SSL certificates).
```